src/App.tsx

```diff
@@ -1,6 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './Pagination';
 
-const App: React.FC = () => <h1>Pagination</h1>;
+const App: React.FC = () => {
+  const [total, setTotal] = useState(42);
+  const [perPage, setPerPage] = useState(5);
+  const [page, setPage] = useState(1);
```

Comment:
I'd rather add this to Pagination component state, cause you obviously don't need lifting state up